From 42306c73cbf1acabb9f8bb4c140b04d5b6f5a23b Mon Sep 17 00:00:00 2001
From: Artur Wieczorek <artwik@wp.pl>
Date: Fri, 3 Apr 2015 21:45:07 +0200
Subject: [PATCH 1/4] Handle wxDateTime data in wxDataViewModel only if
 wxUSE_DATETIME is enabled.

Code using wxDateTime datatype in wxDataViewModel::Compare should be active only if wxUSE_DATETIME is enabled.
---
 src/common/datavcmn.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/common/datavcmn.cpp b/src/common/datavcmn.cpp
index 3fcbf7a..f7455ea 100644
--- a/src/common/datavcmn.cpp
+++ b/src/common/datavcmn.cpp
@@ -343,6 +343,7 @@ int wxDataViewModel::Compare( const wxDataViewItem &item1, const wxDataViewItem
         else if (d1 > d2)
             return 1;
     }
+#if wxUSE_DATETIME
     else if (value1.GetType() == wxT("datetime"))
     {
         wxDateTime dt1 = value1.GetDateTime();
@@ -352,6 +353,7 @@ int wxDataViewModel::Compare( const wxDataViewItem &item1, const wxDataViewItem
         if (dt2.IsEarlierThan(dt1))
             return 1;
     }
+#endif
     else if (value1.GetType() == wxT("bool"))
     {
         bool b1 = value1.GetBool();
-- 
1.9.5.msysgit.0

